extensions/blocks/anchor-fm/editor.js

```diff
@@ -54,20 +56,29 @@ async function setEpisodeTitle( { title } ) {
 	dispatch( 'core/editor' ).editPost( { title } );
 }
 
-const ConvertToAudio = () => (
-	<PluginPostPublishPanel className="anchor-post-publish-outbound-link">
-		<p className="post-publish-panel__postpublish-subheader">
-			<strong>{ __( 'Convert to audio', 'jetpack' ) }</strong>
-		</p>
-		<p>{ __( 'Let your readers listen to your post.', 'jetpack' ) }</p>
-		<p>
-			<a href="https://anchor.fm/wordpress" target="_top">
-				{ __( 'Create a podcast episode', 'jetpack' ) }
-				<Icon icon={ external } className="anchor-post-publish-outbound-link__external_icon" />
-			</a>
-		</p>
-	</PluginPostPublishPanel>
-);
+const ConvertToAudio = () => {
+	useEffect( () => {
+		analytics.tracks.recordEvent( 'jetpack_editor_block_anchor_fm_post_publish_impression' );
```

Comment:
Do we want to add anything to event props here?

Reply:
I don't think we need anything right now.